stats.py

```diff
@@ -94,5 +94,15 @@ def calculate_averages(weather_data):
         avg_max_temp = round(total_max_temp / count)
         avg_min_temp = round(total_min_temp / count)
         avg_mean_humidity = round(total_mean_humidity / count)
-
+        
         return (avg_max_temp, avg_min_temp, avg_mean_humidity)
+
+
+    @staticmethod
+    def find_daily_extremes(weather_data):
+        for data in weather_data:
+            if data.max_temp is not None:
+                print(f"{data.date} Highest: {'+'*data.max_temp} {data.max_temp}")
+            if data.min_temp is not None:
+                print(f"{data.date} Lowest: {'+'*data.min_temp} {data.min_temp}")
```

Comment:
Adding Highest and Lowest was not in the required output. Where is the colors part like red and blue color for + symbols?